aldryn_people/boilerplates/bootstrap3/templates/aldryn_people/group_list.html

```diff
@@ -10,5 +10,11 @@
                 {% include "aldryn_people/includes/person.html" with person=person %}
             {% endfor %}
         {% endfor %}
+        {% if ungrouped_people.count %}
```

Comment:
Very very cosmetic, but the .count could be removed and it would work with objects without .count() :)